Q/qdrant/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+using BinaryBuilder, Pkg
+
+# Collection of pre-build quarto binaries
+name = "qdrant"
```

Comment:
Seems to be spelled
```suggestion
name = "Qdrant"
```
upstream: https://github.com/qdrant/qdrant/blob/a7d21f168097494acc2cbfb2489cfd2da40581ac/README.md.  You also need to rename the directory accordingly